<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QMainWindow" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retrive your Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>621</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>340</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Retrive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>340</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
